packages/utils-connector-tools/tests/requester.ts

```diff
@@ -141,7 +141,7 @@ const testBinaries = async (t: ExecutionContext<RequesterContext>, binType: stri
     t.deepEqual(body.rawContent, content);
 
     // Body should be null
-    t.is(body.content, null);
+    t.is(body.content, '');
```

Comment:
@jackjlynch it appears this change is breaking the test in CI. Looks like `body.content` is the part that isn't typed correctly as it does return `null` at runtime. Ava just glossed over this before as it wasn't enforcing types as thoroughly. Mind taking a look?